templates/base.html.twig

```diff
@@ -1,17 +1,23 @@
 <!DOCTYPE html>
-<html>
+<html lang="fr">
 <head>
     <meta charset="UTF-8">
     <link rel="icon" type="image/png" href="{{ asset('build/images/favicon.png') }}" />
-    <title>{% block title %}Welcome!{% endblock %}</title>
+    <title>{% block title %}Taste{% endblock %}</title>
     {% block stylesheets %}
         {{ encore_entry_link_tags('app') }}
     {% endblock %}
 </head>
 <body>
 
+{% block header %}
+{% endblock %}
+
 {% block body %}{% endblock %}
 
+{% include "_footer.html.twig" %}
+{% block footer %}{% endblock %}
```

Comment:
The _footer.html.twig file is names with a Twig not a twig (and should be with a twig)

Reply:
Resolve name _footer and I use OpenStreetMaps rather than google maps and I rename ViewController with StaticPagesController 😄